packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -168,26 +173,34 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
         return this._inboundSignal;
     }
 
+    // TODO: possibly deprecate
     public get initialSequenceNumber(): number {
         return this.initSequenceNumber;
     }
 
+    // TODO: possibly deprecate
     public get lastSequenceNumber(): number {
+        // This variable now refers to virtual sequence number
         return this.lastProcessedSequenceNumber;
     }
 
+    // TODO: possibly deprecate
     public get lastMessage() {
+        // Potentially refers to virtualized message
         return this.lastProcessedMessage;
     }
 
+    // TODO: possibly deprecate
     public get lastKnownSeqNumber() {
+        // This variable now refers to virtual sequence number
         return this.lastObservedSeqNumber;
     }
 
     public get referenceTerm(): number {
         return this.baseTerm;
     }
 
+    // TODO: possibly deprecate
```

Comment:
I think you want to have some assessment (as part of these comments) on how possible it is.
I think MSN is used all over the place, so that one is not really feasible. 
In other places I'd ask the key question - what do we gain by trying to deprecate it? For example, I think there is no gain of deprecating initialSequenceNumber - that's sequence number of snapshot, runtime has to know virtualized sequence number one way or another. Or maybe other way to say it - for any op that follows, we have to have capability to take sequence number, reference sequence number and virtualize it. And if we have that capability - it's trivial to provide virtualized initialSequenceNumber